tests/core/test_quickadd.py

```diff
@@ -149,6 +149,7 @@ def test_start(self):
         self.assertEqual(expected1, parse(text1))
         self.assertEqual(expected1, parse(text2))
         self.assertEqual(expected1, parse(text3))
+        self.assertEqual(expected2, parse(text4))
```

Comment:
`expected2` and `text4` were unused. Looking at them, I figured it was because of this missing assertion.